### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by dorisiscool** [endink/Mediapipe4u-plugin#113] [General]
Hey, I am trying to build the project and it is reporting error:

Expecting to find a type to be declared in a target rules named 'MediaPipe4U_DemoTarget'.  This type must derive from the 'TargetRules' type defined by Unreal Build Tool.
UnrealBuildTool failed. See log for more details. (C:\Users\Admin\AppData\Roaming\Unreal Engine\AutomationTool\Logs\D+UE_5.1\UBT-MediaPipe4U_Demo-Win64-Development.txt)

Do you know how can I fix it? Thank u!

**Comment by endink** [comment]
Clean your project and rebuild , make sure your UE version is 5.1

**Comment by endink** [comment]
BTW, please use issues section for posting issue, dont post issue in discussion section.

**Comment by endink** [comment]
If problem still occurred,post issue instead discussion.
